Automattic/jetpack
# Jetpack Search: Correct link to language support

Fixes #
This adds a link to the FAQ which documents all the different languages we support


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:
1. Install the Jetpack Search plugin on a test site.
2. Visit the search page in wp-admin
3. Check that the link in the tooltip about languages goes to the FAQ page, instead of just to the current page





<img width="979" alt="Bildschirmfoto 2022-11-07 um 10 26 39" src="https://user-images.githubusercontent.com/19924/200276623-2b8f0a3d-bfda-4825-8a64-1bd3aef9c9b0.png">
